src/Components/CreateImageWizard/formComponents/OscapProfileInformation.tsx

```diff
@@ -29,46 +32,97 @@ const OscapProfileInformation = (): JSX.Element => {
     }
   );
 
+  const enabledServicesDisplayString =
+    oscapProfileInfo?.services?.enabled?.join(' ');
+  const disableServicesDisplayString =
+    oscapProfileInfo?.services?.disabled?.join(' ');
+
   return (
     <>
       {isFetchingOscapProfileInfo && <Spinner size="lg" />}
       {isSuccessOscapProfileInfo && (
-        <TextContent>
-          <br />
-          <TextList component={TextListVariants.dl}>
-            <TextListItem
-              component={TextListItemVariants.dt}
-              className="pf-u-min-width"
-            >
-              Profile description:
-            </TextListItem>
-            <TextListItem component={TextListItemVariants.dd}>
-              {oscapProfileInfo.openscap?.profile_description}
-            </TextListItem>
-          </TextList>
-          <TextList component={TextListVariants.dl}>
-            <TextListItem
-              component={TextListItemVariants.dt}
-              className="pf-u-min-width"
-            >
-              Operating system:
-            </TextListItem>
-            <TextListItem component={TextListItemVariants.dd}>
-              {RELEASES.get(getState()?.values?.['release'])}
-            </TextListItem>
-          </TextList>
-          <TextList component={TextListVariants.dl}>
-            <TextListItem
-              component={TextListItemVariants.dt}
-              className="pf-u-min-width"
-            >
-              Reference ID:
-            </TextListItem>
-            <TextListItem component={TextListItemVariants.dd}>
-              {oscapProfileInfo.openscap?.profile_id}
-            </TextListItem>
-          </TextList>
-        </TextContent>
+        <>
+          <TextContent>
+            <br />
+            <TextList component={TextListVariants.dl}>
+              <TextListItem
+                component={TextListItemVariants.dt}
+                className="pf-u-min-width"
+              >
+                Profile description:
+              </TextListItem>
+              <TextListItem component={TextListItemVariants.dd}>
+                {oscapProfileInfo.openscap?.profile_description}
+              </TextListItem>
+              component={TextListVariants.dl}
```

Comment:
Delete this line please, bug from the V2 wizard version that got copied over (will fix it there in a follow-up PR).